smt/pkg/blockinfo/block_info.go

```diff
@@ -0,0 +1,121 @@
+package blockinfo
+
+import (
+	"github.com/ledgerwatch/erigon/smt/pkg/smt"
+	"math/big"
+)
+
+//func FillReceiptTree(
```

Comment:
hmm, why do we need this commented-out code?

Reply:
WIP - implemented.